app/frontend/components/pages/login/loginPageSidebar.tsx

```diff
@@ -164,23 +167,28 @@ const FileContent = () => (
   </div>
 )
 
-interface Props {
-  authMethod: '' | 'mnemonic' | 'hw-walet' | 'file'
+const SidebarContentByAuthMethod = ({authMethod}: {authMethod: AuthMethodType}) => {
+  switch (authMethod) {
+    case null:
```

Comment:
hmm, shouldn't AuthMethodType be optional? I would expect typescript checks to catch it

Reply:
Same as with setting AuthMethodType to null in the initial state. Perhaps I'd type the auth method as 
`authMethod: AuthMethodType | null` to be more precise